.travis.yml

```diff
@@ -16,7 +16,6 @@ before_install:
 - docker build -f dockerfiles/gomoderator/Dockerfile .
 script:
 - set -e
-- scripts/check_config.sh
```

Comment:
Just curious why we're removing this here? Not a blocker, just trying to remind myself what this does.

Reply:
this verifies that the transformers in the exporters match the transformers in the config. In this case, there's a conflict because the config includes some event transformers that aren't in the exporter. The reason for the discrepancy is that those event transformers need to be in the config for the plugin integration tests to run/pass, but we only want to run storage transformers with this branch